applications/browser/package.json

```
@@ -51,6 +51,11 @@
     }
   },
   "dependencies": {
+    "@theia/ai-core": "1.70.0",
+    "@theia/ai-openai": "1.70.0",
+    "@theia/vsx-registry": "1.70.0",
+    "@theia/plugin-ext": "1.70.0",
```

Biome (2.4.15)

[error] The key `@theia/plugin-ext` was already declared.

(lint/suspicious/noDuplicateObjectKeys)